source/utilities.tex

```diff
@@ -6873,14 +6873,14 @@
     shared_ptr<T> allocate_shared(const A& a, const remove_extent_t<T>& u);     // \tcode{T} is \tcode{U[N]}
 
   template<class T>
-    shared_ptr<T> make_shared_default_init();                                   // \tcode{T} is not \tcode{U[]}
+    shared_ptr<T> make_shared_for_overwrite();                                   // \tcode{T} is not \tcode{U[]}
   template<class T, class A>
-    shared_ptr<T> allocate_shared_default_init(const A& a);                     // \tcode{T} is not \tcode{U[]}
+    shared_ptr<T> allocate_shared_for_overwrite(const A& a);                     // \tcode{T} is not \tcode{U[]}
 
   template<class T>
-    shared_ptr<T> make_shared_default_init(size_t N);                           // \tcode{T} is \tcode{U[]}
+    shared_ptr<T> make_shared_for_overwrite(size_t N);                           // \tcode{T} is \tcode{U[]}
   template<class T, class A>
-    shared_ptr<T> allocate_shared_default_init(const A& a, size_t N);           // \tcode{T} is \tcode{U[]}
+    shared_ptr<T> allocate_shared_for_overwrite(const A& a, size_t N);           // \tcode{T} is \tcode{U[]}
```

Comment:
```suggestion
    shared_ptr<T> allocate_shared_for_overwrite(const A& a, size_t N);          // \tcode{T} is \tcode{U[]}
```
The same applies to all 7 comments above.

Reply:
If you have a sufficiently Unix-y system, please run ../tools/check.sh manually to detect such misaligned comments. (Travis is turned off right now, so we don't get an automatic check.)

Reply:
@jensmaurer the check for comments in check.sh doesn't work on a mac, I had to comment it out of the version I have